src/R/Editor/Application.Test/Completion/IntellisenseTest.cs

```diff
@@ -487,6 +487,31 @@ public class IntellisenseTest : FunctionIndexBasedTest {
         }
 
         [Test]
+        public async Task R_VariablesIndexerMemberCompletion03() {
+            using (var script = await _editorHost.StartScript(ExportProvider, RContentTypeDefinition.ContentType, Workflow.RSessions)) {
+                await ExecuteRCode(_editorHost.HostScript.Session,
+@"
+d <- list(A = c(1:10), B = c(1:10))
```

Comment:
Test needs to use different member names for `d` and `e` to ensure that the right completion list is returned for `e$A$`.